src/renderer/screens/manager/DeviceStorage/index.js

```diff
@@ -24,17 +24,34 @@ import IconTriangleWarning from "~/renderer/icons/TriangleWarning";
 import IconCheckFull from "~/renderer/icons/CheckFull";
 
 import nanoS from "./images/nanoS.png";
+import nanoSDark from "./images/nanoS_dark.png";
```

Comment:
The images seem to be of different sizes, did you export x3 from lottie in all cases?
We don't want shitty rendering on retina

Reply:
I didn't export from lottie but from Figma.
I don't have the 4x retina rendering... So if it's shitty like this I can try another export too

Reply:
Yeah I meant figma sorry, but export in x3 and make sure all images are large. I'm sure the smalls ones in your commit are not going to show good on mac.